charts/firefly/values.yaml

```diff
@@ -402,16 +414,19 @@ dataexchange:
 erc1155:
   enabled: false
 
-  # The Ethconnect URI representing the ERC1155 tokens contract
+  # The address of the deployed ERC1155 contract to use
   contractAddress: "/contracts/erc1155Address"
 
+  # The Ethconnect URI representing the ERC1155 tokens contract
+  ethconnectInstance: "/contracts/erc1155"
+
   # The Ethconnect topic to use for tokens event subscriptions
   ethconnectTopic: "tokensERC1155"
 
   image:
     repository: ghcr.io/hyperledger/firefly-tokens-erc1155
     pullPolicy: IfNotPresent
-    tag: v1.1.5
+    tag: v1.3.0-rc.2
```

Comment:
v1.3.0 please, we might be in 1.3.2 even

Reply:
https://github.com/hyperledger/firefly-helm-charts/pull/76/commits/205f20754d1f00a68130ca2234e50bf3db86aca8